### PR TITLE
KDN-2115 and KDN-2030

### PR DESCRIPTION
**Pull request by Tejas-Waje**

### CODE REVIEW

File: resources/lang/af-ZA/notifications.php [hunk 1]
```diff
@@ -62,17 +62,17 @@
     'schedule' => [
         'new' => [
             'mail' => [
-                'subject' => 'New Schedule Created',
-                'content' => ':name was scheduled for :date',
-                'title'   => 'A new scheduled maintenance was created.',
+                'subject' => ':app_name : DSH Maintenance window announcement',
+                'content' => ':name is scheduled for :date',
+                'title'   => 'A new DSH maintenance schedule is announced.',
```
Comment (kiefersmith) [inline resources/lang/af-ZA/notifications.php:67]: "is announced" sounds off.

File: resources/lang/ar-SA/notifications.php [hunk 2]
```diff
@@ -62,17 +62,17 @@
     'schedule' => [
         'new' => [
             'mail' => [
-                'subject' => 'New Schedule Created',
-                'content' => ':name was scheduled for :date',
-                'title'   => 'A new scheduled maintenance was created.',
+                'subject' => ':app_name : DSH Maintenance window announcement',
+                'content' => ':name is scheduled for :date',
+                'title'   => 'A new DSH maintenance schedule is announced.',
```
Comment (kiefersmith) [inline resources/lang/ar-SA/notifications.php:67]: I would write "has been announced" instead of "is announced".

**Review by kiefersmith — COMMENTED** [review]
I would change all instances of "is announced" to "has been announced".  Maybe "is announced" makes sense in context, not sure.

**Comment by Tejas-Waje** [comment]
I agree, but, the change was specifically requested by AJ
https://klarrio.atlassian.net/browse/KDN-2115?focusedCommentId=22528

**Review by rpartapsing — CHANGES_REQUESTED** [review]
@Tejas-Waje you want to merge into 2.4? Better to create a new version no?
This repo is forked. I think if you push you to 2.4 this is for all cachet users publicly

**Comment by Tejas-Waje** [comment]
> @Tejas-Waje you want to merge into 2.4? Better to create a new version no?
> This repo is forked. I think if you push you to 2.4 this is for all cachet users publicly

Thanks, @rpartapsing, created a new branch [2.4.6](https://github.com/Klarrio/Cachet/tree/2.4.6) from [2.4.5.2](https://github.com/Klarrio/Cachet/tree/2.4.5.2) and changed the base of this PR to [2.4.6](https://github.com/Klarrio/Cachet/tree/2.4.6).
Also, I plan to create a release from it.

**Review by rpartapsing — APPROVED** [review]

**Pull request merged**